common/app/model/content.scala

```diff
@@ -267,6 +267,14 @@ class Content protected (val apiContent: ApiContentWithMeta) extends Trail with
     }
   }
 
+// This is a fix for the hanging letters on long section/series labels on the leftCol breakpoint.
```

Comment:
I would put a more thorough comment in here, preferably linking to an issue on GitHub with screenshots. I really don't like things like this existing in our codebase, but if they have to they need to be clearly explained for future maintainers.